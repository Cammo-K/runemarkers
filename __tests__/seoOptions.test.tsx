import '@testing-library/jest-dom';
import { render } from '@testing-library/react';

import nextSeoOptions from '@/api/seoOptions';
import { DefaultSeo } from 'next-seo';

describe(`nextSeoOptions`, () => {
  it(`should render correctly`, () => {
    const { container } = render(<DefaultSeo {...nextSeoOptions} />);
    expect(container).toMatchSnapshot();
  });
});
