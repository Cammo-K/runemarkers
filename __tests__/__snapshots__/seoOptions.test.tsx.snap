// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nextSeoOptions should render correctly 1`] = `<div />`;
